docs/using-the-jdbc-driver/UsingTheJdbcDriver.md

```diff
@@ -108,7 +108,6 @@ The AWS JDBC Driver has several built-in plugins that are available to use. Plea
 |------------------------------------------------------------------------------------------------|---------------------------|------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | [Failover Connection Plugin](./using-plugins/UsingTheFailoverPlugin.md)                        | `failover`                | Aurora                 | Enables the failover functionality supported by Amazon Aurora clusters. Prevents opening a wrong connection to an old writer node dues to stale DNS after failover event. This plugin is enabled by default.                                                                                                                                                                                                                                                                                                           |
 | [Host Monitoring Connection Plugin](./using-plugins/UsingTheHostMonitoringPlugin.md)           | `efm`                     | Aurora                 | Enables enhanced host connection failure monitoring, allowing faster failure detection rates. This plugin is enabled by default.                                                                                                                                                                                                                                                                                                                                                                                       |
-| Aurora Host List Connection Plugin                                                             | `auroraHostList`          | Aurora                 | Retrieves Amazon Aurora clusters information. <br><br>**:warning:Note:** this plugin does not need to be explicitly loaded if the failover connection plugin is loaded.                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
Was there a chance previously of someone not using the right dialect but then using this plugin for the Host list?

Reply:
Unlikely but may happen

Reply:
is it something we want to now make impossible?

Reply:
Could you please clarify?

Reply:
I was just wondering if there were people using the plugin without configuring the dialect properly.
If that was the case, this change will break their workflow. Hence the question, did we want that to be possible, or we should now force them to use the correct dialect.
If people were using the plugin with the wrong dialect, we need to ensure the documentation is there for them to fix their workflow.